bin/sparkling-env.sh

```diff
@@ -47,7 +47,9 @@ FAT_JAR="sparkling-water-assembly_$SCALA_VERSION-$VERSION-all.jar"
 FAT_JAR_FILE="$TOPDIR/assembly/build/libs/$FAT_JAR"
 major_version=`echo $VERSION | cut -d . -f 1,2`
 version_without_snapshot=`echo $VERSION | cut -d - -f 1`
-PY_EGG="h2o_pysparkling_${major_version}-${version_without_snapshot}-py2.7.egg"
+
+PYTHON_VERSION=`python -c 'import sys; print(".".join(str(x) for x in sys.version_info[0:2]))'`
```

Comment:
We should do this also on windows script equivalents to be consistent.

Maybe it would be nice to print an explanation what happened in case the script couldn't find desired python egg. The user could have changed the python version and I can imagine he/she would be confused when script which they have been using before doesn't work.